### PR TITLE
Make disabled class name on prev/next buttons configurable

### PR DESCRIPTION
**Pull request by gglnx**
The active class is also configurable.

### CODE REVIEW

File: src/lory.js [hunk 1]
```diff
@@ -134,7 +134,9 @@ export function lory (slider, opts) {
             rewind,
             rewindSpeed,
             ease,
-            classNameActiveSlide
+            classNameActiveSlide,
+            classNameDisabledNextCtrl = 'disabled',
```
Comment (jacobjuul) [inline src/lory.js]: Couldn't this be something like `classNameDisabledCtrl` ? (instead of having two variables)

Comment (gglnx) [reply src/lory.js]: It's more flexible, particularly in BEM context: `next-button--disabled` && `prev-button--disabled`.

**Comment by meandmax** [comment]
@gglnx Could you also make a build with current master? Then I would merge this.

**Comment by gglnx** [comment]
@meandmax Ready.

**Pull request merged**